src/catalog/src/system_schema/information_schema/region_statistics.rs

```diff
@@ -57,6 +58,7 @@ const INIT_CAPACITY: usize = 42;
 /// - `table_id`: The table id.
 /// - `region_number`: The region number.
 /// - `region_rows`: The number of rows in region.
+/// - `write_bytes`: The written bytes of the region.
```

Comment:
What does this field represent? Is it the total bytes written in this region, or the incremental value for a single time window?

Reply:
The total bytes written of the region since datanode startup.

Reply:
Since region open or DataNode start? I think they’re different.

Reply:
What happens if a region reopens after migration?

Reply:
Yes, `Since region open` is more accurate.